<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ForecastWidget</class>
 <widget class="QWidget" name="ForecastWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="day2">
   <property name="geometry">
    <rect>
     <x>197</x>
     <y>10</y>
     <width>189</width>
     <height>290</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="hLabel_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>High:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lLabel_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>230</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Low:</string>
    </property>
   </widget>
   <widget class="QLabel" name="highLabel_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>200</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>0f</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lowLabel_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>230</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>0f</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="dateLabel_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>189</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>DATE</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSvgWidget" name="dayIcon_2" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>189</width>
      <height>189</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 0, 0, 0);</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="day3">
   <property name="geometry">
    <rect>
     <x>394</x>
     <y>10</y>
     <width>189</width>
     <height>290</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="hLabel_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>High:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lLabel_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>230</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Low:</string>
    </property>
   </widget>
   <widget class="QLabel" name="highLabel_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>200</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>0f</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lowLabel_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>230</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>0f</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="dateLabel_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>189</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>DATE</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSvgWidget" name="dayIcon_3" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>189</width>
      <height>189</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 0, 0, 0);</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="day4">
   <property name="geometry">
    <rect>
     <x>591</x>
     <y>10</y>
     <width>189</width>
     <height>290</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="hLabel_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>High:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lLabel_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>230</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Low:</string>
    </property>
   </widget>
   <widget class="QLabel" name="highLabel_4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>200</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>0f</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lowLabel_4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>230</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>0f</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="dateLabel_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>189</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>DATE</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSvgWidget" name="dayIcon_4" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>189</width>
      <height>189</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 0, 0, 0);</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="day1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>189</width>
     <height>290</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="hLabel_1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>High:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lLabel_1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>230</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Low:</string>
    </property>
   </widget>
   <widget class="QLabel" name="highLabel_1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>200</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>0f</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lowLabel_1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>230</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>0f</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="dateLabel_1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>189</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>DATE</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSvgWidget" name="dayIcon_1" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>189</width>
      <height>189</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 0, 0, 0);</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QSvgWidget</class>
   <extends>QWidget</extends>
   <header location="global">qsvgwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
